steamcmd/ubuntu/Dockerfile

```diff
@@ -25,5 +25,5 @@ USER        container
 ENV         USER=container HOME=/home/container
 WORKDIR     /home/container
 
-COPY        ./entrypoint.sh /entrypoint.sh
+COPY        ./../entrypoint.sh /entrypoint.sh
```

Comment:
Why is this `./../` and no just `../`?

Reply:
> Why is this `./../` and no just `../`?

i think, because of this (as example)

https://github.com/parkervcp/yolks/blob/6c1690ed34a0d018ce342a511cc11a032b2ab3f1/java/19/Dockerfile#L16